analyzers/its/expected/.gitattributes

```diff
@@ -1,2 +1,2 @@
-# Text files should have CRLF line endings, even on non-Windows machines
-*.json eol=crlf
+# Force LF to avoid "git diff" warning about replacing CRLF
+*.json eol=lf
```

Comment:
_Hint:_ The original `.gitignore` from the root was reverted. This is a change of another file inside `/expected` folder.